templates/influxdb_dropin.epp

```diff
@@ -1,4 +1,5 @@
-<%- | String $cert, String $key | -%>
+<%- | String $cert, String $key, Integer $port | -%>
```

Comment:
Instead of using Integer, can you switch to https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/port/unprivileged.pp ?

Reply:
Done :+1: